content/HowToCreateBitbakeRecipeLocSrc.md

```diff
@@ -0,0 +1,41 @@
+<img src="../files/img/2018_MistyWest_LogoCombo_FINAL_RGB.png" alt="MistyWest" width="400"/><div style="text-align: right">back to [index](../README.md)</div>
+
+# How to create a custom bitbake recipe from local sources
+
+Here is an example recipe that you can use:
+```
+SUMMARY = "Recipe for DesiredApp"
+LICENSE = "CLOSED"
+
+SRC_URI = "file://path/to/DesiredApp"
+
+S = "${WORKDIR}"
+
+do_compile() {
+    ${CC} ${CFLAGS} ${LDFLAGS} ${WORKDIR}/DesiredApp.c -o ${WORKDIR}/DesiredApp
+}
+
+do_install() {
+    install -d ${D}${bindir}
+    install -m 0755 ${WORKDIR}/DesiredApp ${D}${bindir}
+}
```

Comment:
The point of `inherit cmake` is not to override the `do_compile` and `do_install` functions.
So I think it is best to say if you don't have one already, please use a build tool.

And honestly, the only difference between this approach and the Git approach is the `SRC_URI`, `SRCREV`, and `S` variables. The rest of it is the same. So maybe we can consolidate them into one tutorial?